test/scalability/README.md

```diff
@@ -0,0 +1,64 @@
+# Scalability test
+
+Is a test meant to detect regressions int the Kueue's overall scheduling capabilities. 
```

Comment:
We can but let's come up with a precise naming scheme, not only for the code location but also the artifacts and make targets, otherwise it will be hard to follow the terminology.

Reply:
Yes, the target names should match. Maybe we can put all the targets for "performance" inside its own Makefile. so it would look like:

`make test/performance/jobs test/performance/scheduling`

But we can leave them for a follow up.

Reply:
Let's go with the follow-up then